CHANGELOG.md

```diff
@@ -42,6 +45,12 @@ This changelog documents all notable changes to VAST and is updated on every rel
 - VAST does not lose query capacity when backlogged queries are cancelled any more.
   [#2092](https://github.com/tenzir/vast/pull/2092)
 
+- Correctly adjust the index statistics when applying partition transforms.
+  [#2097](https://github.com/tenzir/vast/pull/2097)
+
+- Fixed a use-after-move bug that could have resulted in incorrect offset ranges being returned from the meta index.
```

Comment:
While being a bug-fix, this not something the user can understand. Can we explain the *effect* that this bug has in user lingo?